app/AppCache.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+use Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache;
+
+class AppCache extends HttpCache
+{
+  class HogeClass {
+    Class HogeHogeClass {
```

Comment:
__[development][code_sniffer]__
* Line indented incorrectly; expected 8 spaces, found 4
* Multiple classes defined in a single file
* Missing class doc comment
* Each class must be in a namespace of at least one level (a top-level vendor name)
* PHP keywords must be lowercase; expected "class" but found "Class"
* Opening brace of a class must be on the line after the definition